### PR TITLE
docs(start-here,core-concepts,development,import-models,): restructure core-concepts/model

### PR DESCRIPTION
**Pull request by xiaofei-du**
Because

- we're iterating the documentation

This commit

- restructure `core-concepts/model`
- update sidebar titles
- add more bullet points in `Learn VDP` on `start-here/getting-started` page
- add contents in `start-here/instill-cloud`
- refactor here and there

### CODE REVIEW

File: src/pages/docs/development/setup-local-development.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { DocsLayout } from "@/components/docs";
 export const meta = {
-  title: "Local development",
+  title: "Set up local development",
```
Comment (pinglin) [inline src/pages/docs/development/setup-local-development.mdx:3]:
```suggestion
  title: "Set up local development environment",
```

File: src/pages/docs/core-concepts/model.mdx [hunk 2]
```diff
@@ -103,33 +134,12 @@ http://localhost:8083/v1alpha/models/{model-id}/instances/{instance-id}:test-mul
...
-<CH.Code>
-
-```json single-model-instance-in-pipeline
-{
-  "source": "source-connectors/source-http",
-  "model_instances": ["models/model-1/instances/v1.0"],
-  "destination": "destination-connectors/postgres-db"
-}
-```
+## Release stage
```
Comment (pinglin) [inline src/pages/docs/core-concepts/model.mdx:137]: Release stage has been moved to `core-concepts/overview`. Components can share that section directly.

**Review by pinglin — CHANGES_REQUESTED** [review]

File: src/pages/docs/core-concepts/model.mdx (hunk 2 above)
Comment (xiaofei-du) [reply src/pages/docs/core-concepts/model.mdx:137]: updated

File: src/pages/docs/core-concepts/model.mdx [hunk 3]
```diff
@@ -12,43 +12,74 @@ export default ({ children }) => (
...
+
+_VDP captures the versioning of your models via model instances._
+During model development, ML teams version models to track and manage changes and collaborate between members.
+As you use VDP to import your model, different versions of your model are also imported as tagged model instances.
 In this case, you can keep a single history of your models and the original versioning serves as the single source of truth.
 
-_VDP enables model versioning using the concepts of model and model instance_. A model represents an algorithm run on data and it may have multiple model instances.
-Each model instance represents a tagged snapshot of the model. The tags of the instances depend on the model source and what versioning control tool the model source uses.
-Please refer to [Import Models](/docs/import-models/overview) to learn about model versioning with supported model sources.
+Sometimes we use the two terms _model_ and _model instance_ interchangeably.
```
Comment (pinglin) [inline src/pages/docs/core-concepts/model.mdx:36]:
```suggestion
Throughout the documentation, we use the two terms _model_ and _model instance_ interchangeably.
```

File: src/pages/docs/core-concepts/model.mdx [hunk 4]
```diff
@@ -12,43 +12,74 @@ export default ({ children }) => (
...
+For example, we say that model is the core component of data pipeline to convert the unstructured visual data to structured insights.
```
Comment (pinglin) [inline src/pages/docs/core-concepts/model.mdx:37]:
```suggestion
For example, when we say that **Model** is the core component of data pipeline to convert the unstructured visual data to structured insights, what actually happens underneath is the model instances being used to construct a data pipeline.
```

File: src/pages/docs/core-concepts/model.mdx [hunk 5]
```diff
@@ -12,43 +12,74 @@ export default ({ children }) => (
...
+This is a high level concept, but underneath model instances are used to construct a data pipeline.
```
Comment (pinglin) [inline src/pages/docs/core-concepts/model.mdx:38]:
```suggestion
```

File: src/pages/docs/core-concepts/model.mdx [hunk 6]
```diff
@@ -12,43 +12,74 @@ export default ({ children }) => (
...
+The examples below showcase pipeline recipes that incorporate single or multiple model instances.
```
Comment (pinglin) [inline src/pages/docs/core-concepts/model.mdx:39]:
```suggestion
A VDP pipeline can have multiple model instances. The examples below showcase pipeline recipes that incorporate single or multiple model instances.
```

**Review by pinglin — CHANGES_REQUESTED** [review]

**Pull request merged**